hal/src/stm32f2xx/core_hal_stm32f2xx.c

```diff
@@ -240,6 +240,11 @@ void HAL_Core_Config(void)
 #ifdef HAS_SERIAL_FLASH
     //Initialize Serial Flash
     sFLASH_Init();
+#if PLATFORM_ID == PLATFORM_DUO_PRODUCTION
```

Comment:
Please pardon me for having no idea what exactly to do as you told. Could you tell me more details?